Tabloid/client/src/components/comment/PostComment.js

```diff
@@ -0,0 +1,33 @@
+import React, { useContext, useEffect } from "react";
+import { CommentContext } from "../../providers/CommentProvider";
+import { Container, Table } from "reactstrap";
+import Comment from "./Comment";
+import { Link } from "react-router-dom";
+import { useHistory, useParams } from "react-router-dom";
+
+const CommentList = () => {
```

Comment:
If you don't want to get all the post stuff from the comment query you could also possibly get the current post from the post context.

Reply:
Done!